### DISCUSSION TITLE
how to handle geometry errors like parts intersect in maxwell  with pyaedt?

### DISCUSSION THREAD

**Discussion by iaskgithub** [ansys/pyaedt#3848] [Q&A] [answered]
how to handle geometry errors like parts intersect in maxwell  with pyaedt?  when a geometry errors like parts intersect  is occured in maxwell ,how to check with pyaedt? online waiting,thank u

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @iaskgithub ,

There is no specific method to handle intersections, but you can use a similar approach that this method in icepak.py:

https://github.com/ansys/pyaedt/blob/main/pyaedt/icepak.py#L1092

You can see how PyAEDT is parsing the validation logger and it searches for "intersections" and then performs some operations.

Please close the discussion if the answer is useful.

**Reply by iaskgithub** [reply]
thank you[@Samuelopez-ansys](https://github.com/Samuelopez-ansys)
